### PR TITLE
Multilang: add missing Chinese e2e toggle translations

### PR DESCRIPTION
**Pull request by cydia2020**
Chinese (simplified), (traditional), and Japanese

### CODE REVIEW

**Comment by sshane** [comment]
@eFiniLan @ponzu07 how do the traditional and Japanese translations look?

File: selfdrive/ui/translations/main_zh-CHT.ts [hunk 1]
```diff
@@ -1195,12 +1195,12 @@ location set</source>
     <message>
         <location line="+11"/>
         <source>🌮 End-to-end longitudinal (extremely alpha) 🌮</source>
-        <translation type="unfinished"></translation>
+        <translation>🌮 端對端神經網路縱向控制（實驗性功能） 🌮</translation>
```
Comment (ZwX1616) [inline selfdrive/ui/translations/main_zh-CHT.ts:1198]:
```suggestion
        <translation>🌮 端對端縱向控制（實驗性功能） 🌮</translation>
```

File: selfdrive/ui/translations/main_zh-CHT.ts [hunk 2]
```diff
@@ -1195,12 +1195,12 @@ location set</source>
...
     </message>
     <message>
         <location line="+1"/>
         <source>Let the driving model control the gas and brakes, openpilot will drive as it thinks a human would. Super experimental.</source>
-        <translation type="unfinished"></translation>
+        <translation>讓駕駛神經網路直接控製油門和剎車，openpilot 將會模仿人類司機的駕駛方式。早期試用版。</translation>
```
Comment (ZwX1616) [inline selfdrive/ui/translations/main_zh-CHT.ts:1203]:
```suggestion
        <translation>讓駕駛模型直接控製油門和剎車，openpilot將會模仿人類司機的駕駛方式。該功能仍非常實驗性。</translation>
```

File: selfdrive/ui/translations/main_zh-CHS.ts [hunk 3]
```diff
@@ -1193,12 +1193,12 @@ location set</source>
     <message>
         <location line="+11"/>
         <source>🌮 End-to-end longitudinal (extremely alpha) 🌮</source>
-        <translation type="unfinished"></translation>
+        <translation>🌮 端对端神经网络纵向控制（实验性功能） 🌮</translation>
```
Comment (ZwX1616) [inline selfdrive/ui/translations/main_zh-CHS.ts:1196]:
```suggestion
        <translation>🌮 端对端纵向控制（实验性功能） 🌮</translation>
```

File: selfdrive/ui/translations/main_zh-CHS.ts [hunk 4]
```diff
@@ -1193,12 +1193,12 @@ location set</source>
...
     </message>
     <message>
         <location line="+1"/>
         <source>Let the driving model control the gas and brakes, openpilot will drive as it thinks a human would. Super experimental.</source>
-        <translation type="unfinished"></translation>
+        <translation>让驾驶神经网络模型直接控制油门和刹车，openpilot将会模仿人类司机的驾驶方式。该功能仍非常实验性。</translation>
```
Comment (ZwX1616) [inline selfdrive/ui/translations/main_zh-CHS.ts:1201]:
```suggestion
        <translation>让驾驶模型直接控制油门和刹车，openpilot将会模仿人类司机的驾驶方式。该功能仍非常实验性。</translation>
```

Comment (cydia2020) [reply selfdrive/ui/translations/main_zh-CHS.ts]: `該功能仍非常實驗性` sounds wrong and is probably grammatically incorrect imho as it's missing a verb, if you insist on keeping 仍 then it should be 該功能仍非常具有實驗性 or 該功能仍在測試中 @NicoPoier what do you reckon?

File: selfdrive/ui/translations/main_zh-CHS.ts [hunk 5]
```diff
@@ -414,7 +414,7 @@
         <location line="+12"/>
         <source>Get turn-by-turn directions displayed and more with a comma
 prime subscription. Sign up now: https://connect.comma.ai</source>
-        <translation>订阅comma prime以获取导航。 
+        <translation>订阅comma prime以获取导航。
```
Comment (sshane) [inline selfdrive/ui/translations/main_zh-CHS.ts:417]:
```suggestion
        <translation>订阅comma prime以获取导航。 
```

File: selfdrive/ui/translations/main_zh-CHS.ts (hunk 4 above)
Comment (ZwX1616) [reply selfdrive/ui/translations/main_zh-CHS.ts]: this sentence has the same structure as "这家店很不错" and that is correct no?

Comment (cydia2020) [reply selfdrive/ui/translations/main_zh-CHS.ts]: One could argue that 很 is not necessary at all in this sentence, removing it does not change the meaning of the sentence, it is merely a placeholder, but you cannot remove the same part of speech (仍) in the translated sentence.
image

**Pull request merged**